index.bs

```diff
@@ -85,8 +88,8 @@ The DeviceOrientationSensor Interface {#deviceorientationsensor-interface}
 <pre class="idl">
   [Constructor(optional SensorOptions sensorOptions)]
   interface DeviceOrientationSensor : Sensor {
-    void populateQuaternion(Float32Array array);
-    void populateMatrix(Float32Array array);
+    void populateQuaternion(Float32Array buffer);
```

Comment:
What do we do with the timestamps, here?

We need to expose those somehow and tie them to the correct typed array.